branch.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Basic Model</title>
+	<link rel="stylesheet" type="text/css" href="./branch.css">
+</head>
+
+<body onkeydown = "KeyPresser(event)">
+	<div class="map">
+		<img id="sprite" src="file:///Users/IMG_3886.jpg">
```

Comment:
Your file path points to an image file on the user’s machine. Needless to say, you can probably expect that file won’t be there in most cases! Better to add an image file to this project and point to that.